dist/s_docs

```diff
@@ -87,7 +87,21 @@ structurechk()
 spellchk()
 {
     # If aspell has been installed, run a spell check.
-    type aspell > /dev/null 2>&1 || return
+    type aspell > /dev/null 2>&1 || (echo "Cannot find aspell!"; exit 1)
+
+    # Make sure custom spellings are sorted.
+    header="personal_ws-1.1 en 200"
+    custom_spellings=../src/docs/spell.ok
+
+    # We need to remove the required header first.
+    sed -i '/personal_ws-1.1 en 200/d' $custom_spellings
+    if ! sort -c $custom_spellings; then
```

Comment:
I know that `aspell` doesn't work with language `C` but everything else relies on `C` sorting order. So let's set `LC_ALL=C` for `sort` only: `LC_ALL=C sort -c $custom_spellings`.

Or alternatively, we can set globally `LC_ALL=C` in the file but call aspell with `LC_ALL=EN_us aspell ...`.